src/device-manager/cocoa/NLNetworkInfo.mm

```diff
@@ -163,9 +162,8 @@ - (NetworkInfo)toNetworkInfo
     }
 
     if (_ThreadNetworkKey) {
-        networkInfo.ThreadNetworkKeyLen = [_ThreadNetworkKey length];
-        networkInfo.ThreadNetworkKey = (uint8_t *) malloc(networkInfo.ThreadNetworkKeyLen);
-        memcpy(networkInfo.ThreadNetworkKey, [_ThreadNetworkKey bytes], networkInfo.ThreadNetworkKeyLen);
+        networkInfo.ThreadNetworkKey = (uint8_t *) malloc(NetworkInfo::kThreadNetworkKeyLength);
+        memcpy(networkInfo.ThreadNetworkKey, [_ThreadNetworkKey bytes], NetworkInfo::kThreadNetworkKeyLength);
```

Comment:
Thread PSKc seems not to be exported into the Objective C land.

Reply:
Updated